### PR TITLE
#2014  Increase maximum font size option

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:

   - fixes issue #2014
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)

2. `src/main/resources/release_notes.md` ...

 - [x] has been updated
 - [ ] does not require update

3. [Optional] suggested person to review this PR @___

### CODE REVIEW

File: src/main/java/core/gui/comp/panel/DoubleLabelPanel.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ private void addLabels() {
         layout.setVgap(0);
         setLayout(layout);
         // Fix right label width to avoid components moving when values change.
-        var width = 60;
+        var width = UserParameter.instance().fontSize * 5;
```
Comment (tychobrailleur) [inline src/main/java/core/gui/comp/panel/DoubleLabelPanel.java:29]: Not sure whether this helps (actually, most definitely not here, but maybe in other classes), but `stringWidth` in `FontMetrics` can help finding the size of a string in a given font. See for example [here](https://github.com/ho-dev/HattrickOrganizer/blob/master/src/main/java/core/icon/TextIcon.java#L33).

File: src/main/java/module/matches/TeamsRatingPanel.java [hunk 2]
```diff
@@ -98,14 +97,15 @@ private void addListeners() {
...
+		var fontSize = UserParameter.instance().fontSize;
 
 		f =  new JLabel("").getFont();
 		f = f.deriveFont(f.getStyle() | Font.BOLD);
 
 		bars = new JProgressBar[14];
 		for (int i = 0; i < 14; i++) {
 			bars[i] = new JProgressBar(0, 100);
-			bars[i].setPreferredSize(new Dimension(200, 20)); //25 if nimbus
+			bars[i].setPreferredSize(new Dimension(20 * fontSize, 2 * fontSize)); //25 if nimbus
```
Comment (tychobrailleur) [inline src/main/java/module/matches/TeamsRatingPanel.java:108]: Am I right in saying the comment at the end of the line is no longer valid/relevant?

**Review by tychobrailleur — APPROVED** [review]
Sweet, a lot of good stuff in there!  I have added some comments, but none blocking IMHO.

File: src/main/java/core/gui/comp/panel/DoubleLabelPanel.java (hunk 1 above)
Comment (wsbrenk) [reply src/main/java/core/gui/comp/panel/DoubleLabelPanel.java:29]: Good idea for future refactorings. For time being, i only wanted to replace any constant by font size related values.

File: src/main/java/module/matches/TeamsRatingPanel.java (hunk 2 above)
Comment (wsbrenk) [reply src/main/java/module/matches/TeamsRatingPanel.java:108]: no idea, what is the reason for that comment. i will remove it.

**Pull request merged**